src/Http/Livewire/LivewireDatatable.php

```diff
@@ -470,17 +475,22 @@ public function sessionStorageKey()
 
     public function getSessionStoredSort()
     {
-        if (! $this->persistSort) {
+        if (!$this->persistSort) {
+            return;
+        }
+
+        if (!$this->multisort) {
+            $this->sort = session()->get($this->sessionStorageKey() . $this->name . '_sort', $this->sort);
             return;
         }
 
-        $this->sort = session()->get($this->sessionStorageKey() . $this->name . '_sort', $this->sort);
-        $this->direction = session()->get($this->sessionStorageKey() . $this->name . '_direction', $this->direction);
+        session()->put([$this->sessionStorageKey() . $this->name . '_multisort' => implode(',', $this->sort)]);
```

Comment:
@kountouris7 Why are we putting something in the session in a method called `getSessionStoredSort`? It's a bit confusing that for simple sort we get something from the session but with multisort we put something in the session.

Reply:
This a wrong. It should have been get instead. Fixing now.